.github/workflows/master.yml

```diff
@@ -44,3 +44,63 @@ jobs:
           TNB_SECRET_KEY: dummy
           TNB_NODE_SIGNING_KEY: ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
 # QUALITY-ASSURANCE-END
+
+  publish-image:
+    name: Publish docker images
+    needs: quality-assurance
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to GitHub Docker Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: $GITHUB_ACTOR
+          password: ${{ secrets.NODE_GITHUB_TOKEN }}
+      - name: Push node image to Github Packages
+        uses: docker/build-push-action@v2
+        with:
+          target: node
+          tags: ghcr.io/thenewboston-developers/node:latest
+          push: true
+          # TODO(dmu) LOW: Make a better way of forcing the download of latest blockchain state
```

Comment:
what is this TODO about?

Reply:
@olegtropinin please, address this comment. Need rewording or removal. I guess this is related with downloading the blockchain state into image for node discovery algorithm. This now will be done with https://thenewboston.atlassian.net/browse/BC-184